src/scripts/models/content.coffee

```diff
@@ -50,7 +50,7 @@ define (require) ->
     defaultPage: ->
       result = 1
       if @isBook() and @isCcap()
-        result += 1 while @getPage(result).get('title').match(/^Preface/)
+        result += 1 while @getPage(result).get('title').match(/Preface/g)
```

Comment:
You can remove the `g` from the regex `/Preface/g`. This modifier makes a global search, so it finds all matches. Without it, the search stops at first match.
It shouldn't make a difference though, because each book has only one Preface.